### PR TITLE
Renaming to Event Controller

### PR DESCRIPTION
**Pull request by etsauer**
@redhat-cop/casl

### CODE REVIEW

File: event-controller-template.yml [hunk 1]
```diff
@@ -160,11 +160,11 @@ parameters:
 - description: The name for the application.
   name: APPLICATION_NAME
   required: true
-  value: event-watcher
+  value: event-controller
 - description: Source code repo URL
   name: SOURCE_CODE_URL
   required: true
-  value: https://github.com/redhat-cop/event-watcher.git
+  value: https://github.com/redhat-cop/event-controller.git
```
Comment (oybed) [inline event-controller-template.yml]: should be `openshift-event-controller.git`

File: event-controller-template.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Template
 labels:
-  template: event-watcher
+  template: event-controller
```
Comment (oybed) [inline event-controller-template.yml:4]: do we want to spell it out to match the repo - i.e.; `openshift-event-controller`?

Comment (oybed) [reply event-controller-template.yml:4]: for all of the template content ...

**Review by oybed — CHANGES_REQUESTED** [review]

File: event-controller-template.yml (hunk 1 above)
Comment (oybed) [reply event-controller-template.yml]: BTW: this one in particular is a required change

File: event-controller-template.yml (hunk 2 above)
Comment (etsauer) [reply event-controller-template.yml:4]: @oybed to me, once you're in openshift, using `openshift` in the name of things becomes redundant.

**Comment by oybed** [comment]
@etsauer testing done and content looking good. Merging.

**Review by oybed — APPROVED** [review]

**Pull request merged**